libs/books/feature/src/lib/reading-list/reading-list.component.html

```diff
@@ -8,15 +8,34 @@
       <div class="reading-list-item--details--author">
         {{ b.authors.join(',') }}
       </div>
+      <div *ngIf="(b.finished)" class="finished" data-testing="reading-list-finished">
+        <mat-icon class="finished">done</mat-icon>Finished on {{b.finishedDate | date: 'dd/MM/yyyy'}}
+      </div>
     </div>
     <div>
-      <button
-        mat-icon-button
-        color="warn"
-        (click)="removeFromReadingList(b)"
-      >
-        <mat-icon attr.aria-label="Remove {{b.title}} from reading list">remove_circle</mat-icon>
-      </button>
+      <mat-grid-list cols="2" rowHeight="1:1">
+        <mat-grid-tile>
+          <button
+            mat-icon-button
+            color="primary"
+            [attr.aria-label]="'Mark ' + b.title + ' as read'"
+            (click)="markAsRead(b)"
+            *ngIf="(!b.finished)"
+            data-testing="reading-list-mark-as-read"
+          >
+            <mat-icon>done</mat-icon>
+          </button>
+        </mat-grid-tile>
+        <mat-grid-tile>
+          <button
+            mat-icon-button
+            color="warn"
+            (click)="removeFromReadingList(b)"
+          >
+            <mat-icon attr.aria-label="Remove {{b.title}} from reading list">remove_circle</mat-icon>
```

Comment:
Again, inconsistency in your attribute binding syntax...

Reply:
Interpolated the aria-labels and moved them to the <button>